packages/swapper/src/swappers/thorchain/utils/bitcoin/utils/getThorTxData.ts

```diff
@@ -46,14 +45,16 @@ export const getThorTxInfo: GetBtcThorTxInfo = async ({
       `${deps.midgardUrl}/thorchain/inbound_addresses`
     )
 
-    const btcInboundAddresses = inboundAddresses.find((inbound) => inbound.chain === 'BTC')
+    const sellAssetInboundAddresses = inboundAddresses.find(
+      (inbound) => inbound.chain === sellAsset.symbol.toUpperCase()
```

Comment:
I know this was not introduced in this PR, but what about renaming the `inboundAddress`/es terminology? It doesn't really refer to an inbound *address* if you're accessing some properties on it like chain or... address.
What about `inboundAddressData` ? Similarly, `InboundResponse[]` seems wrong to me. A reponse is a singular entity, and `Response[]` would semantically refer to a list of XHR responses, which isn't the case, so I would rename `InboundResponse` to `InboundAddressData` so that the typed response is `InboundAddressData[]`, which makes sense as a list.

Reply:
I see what your saying but I dont feel very strongly either way about this. Id prefer to make the changes in another PR tho

Reply:
A good workflow I like to use when tracking comments to be addressed in a future PR is the "Reference in new issue" button. Then you can just resolve the comment with a link to the new ticket in-line, and no context is lost 👌 

<img width="314" alt="Screen Shot 2022-08-08 at 5 16 38 pm" src="https://user-images.githubusercontent.com/97164662/183361252-df7234c8-c6e8-4986-b1b9-2d52c31122fe.png">

Reply:
Totally agree with this belonging in another PR to keep things atomic, and thanks ser @0xApotheosis, that sounds a lot easier than doing this manually 🪄 !